index.html

```diff
@@ -68,6 +68,177 @@ <h1 class="main__title">I'm Alexandra. <br> Glad to see you!</h1>
     </main>
     <!-- END -->
 
+    <!-- WORKS SECTION -->
+    <section id="portfolio" class="work__section">
+        <article class="work__description">
+            <figure class="work__image__container">
+                <img src="/assets/work1.png" alt="first work image" class="work__image">
+            </figure>
+            <div class="work__features__container">
+                <h2 class="work__title">Tonic</h2>
+                <ul class="work__features">
+                    <li class="work__features__item">CANOPY</li>
+                    <li class="work__features__item">Back End Dev</li>
+                    <li class="work__features__item">2015</li>
+                </ul>
+                <p class="work__paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work__list__container">
+                    <li class="work__language__item">html</li>
+                    <li class="work__language__item">css</li>
+                    <li class="work__language__item">javascript</li>
+                </ul>
+                <button class="work__button">See Project</button>
```

Comment:
[Optional] Great work on implementing the BEM CSS architecture! I noticed that the button, such as the "See project" button, is currently using a <button> tag. It is important to ensure that the button follows proper button styling, including the active, hover, and pressed states as specified in the design. By ensuring the button adheres to the correct styling and behavior, you can provide a consistent and user-friendly experience for your visitors

Please apply the necessary CSS styles to create the active, hover, and pressed states as indicated in the design specifications. :pray:

![image](https://github.com/Feliperto/PortfolioWebsite-Microverse/assets/55840999/a6218e05-ee73-4b74-bc1b-b53a669f7476)

**Note: All Buttons should have proper styles as in the desinge.**